src/libraries/icubmod/embObjMotionControl/eomcParser.cpp

```diff
@@ -2386,3 +2327,39 @@ void JointsSet::dumpdata(void)
 
 }
 
+bool Parser::checkJointTypes(PidInfo *pids, std::string pid_type)
```

Comment:
This ought to be `const std::string& pid_type`.

Reply:
done!

Reply:
Same comment here.

Reply:
fixed!